src/main/webapp/assignPanel.js

```diff
@@ -268,5 +261,6 @@ function assignDisplayNextEmail() {
     return;
   }
 
-  subjectLineElement.innerText = actionableEmails[0].subject;
+  subjectLineElement.innerText =
+      `(From: ${actionableEmails[0].sender}) ${actionableEmails[0].subject}`;
```

Comment:
Thanks for looking into this!
Though, now it shows "Subject: From: ...". I wonder if we should change the "Subject" label, but I'm not particularly concerned.

Reply:
I think it should be fine, as it now looks like: Subject: (From: Austin Teshuba) ACTION REQUIRED...

I see what your saying though, it's just that space is a premium. I can't think of anything better right now, so I'll leave it for now and we can change this later (it'll be a 1 line PR)